packages/rollup-plugin-replace-with-ast/src/index.ts

```diff
@@ -0,0 +1,81 @@
+import {createFilter} from '@rollup/pluginutils';
+import {
+  parse,
+  Node,
+  ImportDeclaration,
+  ExportNamedDeclaration,
+  ExportAllDeclaration,
+  Program,
+} from 'acorn';
+import MagicString from 'magic-string';
+
+interface PluginOptions {
+  include?: string | string[];
+  exclude?: string | string[];
```

Comment:
Why add those `include` `exclude` options if we don't use them ?

Reply:
they do get used in the code